src/app/modules/feedback/feedback.service.ts

```diff
@@ -111,3 +111,27 @@ export const getFormFeedbacks = (
     }
   })
 }
+
+export const checkHasPreviousFeedback = (
```

Comment:
can we add jsdocs for all new functions? also I think this function name conveys exactly what this function is doing. maybe something like `hasPrevFeedbackForSubmission`? usually we prefix functions which return booleans with `has` or `does`

Reply:
Updated the PR to add jsdocs, and rename the functions